src/net.rs

```diff
@@ -33,19 +35,53 @@ pub const TELEGRAM_API_URL: &str = "https://api.telegram.org";
 ///
 /// If `TELOXIDE_PROXY` exists, but isn't correct url.
 pub fn client_from_env() -> reqwest::Client {
-    use crate::bot::{sound_bot, TELOXIDE_PROXY};
     use reqwest::Proxy;
 
-    let builder = sound_bot();
+    const TELOXIDE_PROXY: &str = "TELOXIDE_PROXY";
+
+    let builder = default_reqwest_settings();
 
     match std::env::var(TELOXIDE_PROXY).ok() {
-        Some(proxy) => builder.proxy(Proxy::all(&proxy).expect("creating reqwest::Proxy")),
+        Some(proxy) => builder.proxy(Proxy::all(&proxy).expect("reqwest::Proxy creation failed")),
         None => builder,
     }
     .build()
     .expect("creating reqwest::Client")
 }
 
+/// Returns a reqwest client builder with default settings.
+///
+/// Client built from default settings is supposed to work in long time
+/// durations, see the [issue 223].
+///
+/// Current setting are:
+/// - `connection/keep-alive` default header
+/// - connection timeout of 5 seconds
+/// - timeout of 17 seconds
+/// - tcp_nodelay is on
```

Comment:
```suggestion
/// The current settings are:
///  - The `connection/keep-alive` default header.
///  - A connection timeout of 5 seconds.
///  - A timeout of 17 seconds.
///  - `tcp_nodelay` is on.
```

Reply:
I don't really like this style (lists consisting of full sentences), but whatever. Can you update [`CODE_STYLE.md`](https://github.com/teloxide/teloxide/blob/dev/CODE_STYLE.md) to enforce the style?